content/influxdb/cloud-dedicated/query-data/execute-queries/optimize-queries.md

```diff
@@ -29,7 +31,25 @@ Use the following tools to help you identify performance bottlenecks and trouble
   - [Inspect Flight response headers](#inspect-flight-response-headers)
 - [Retrieve query information](#retrieve-query-information)
 
-<!-- /TOC -->
+
+## Strategies for improving query performance
+
+A query may be slow due to the following reasons:
+
+- It queries a large time-range of data.
+- It includes intensive operations, such as `ORDER BY` on large amounts of data.
+- The query plan isn't optimal--for example, applying the same sort (`ORDER BY`) to already sorted data.
+- It needs to retrieve many parquet files from object storage. The same query performs better if it retrieves fewer - though, larger - files.
+- It queries many overlapped parquet files.
```

Comment:
Similar to the above, these can only see in the the query plan, I suggest you mention these here briefly add a link to show users how to know when these happen